code/modules/projectiles/guns/ballistic/bow.dm

```diff
@@ -0,0 +1,65 @@
+/obj/item/gun/ballistic/bow
+	name = "wooden bow"
+	desc = "some sort of primitive projectile weapon. used to fire arrows."
+	icon_state = "bow"
+	item_state = "bow"
+	w_class = WEIGHT_CLASS_BULKY
+	weapon_weight = WEAPON_HEAVY //need both hands to fire
+	force = 5
+	mag_type = /obj/item/ammo_box/magazine/internal/bow
+	fire_sound = 'sound/weapons/bowfire.ogg'
+	slot_flags = ITEM_SLOT_BACK
+	item_flags = NONE
+	casing_ejector = FALSE
+	internal_magazine = TRUE
+	pin = null
+	no_pin_required = TRUE
```

Comment:
this may break some shit

Reply:
Ive tested every gun by firing them and then removing their pin and trying to fire again.
no gun that was not a debug blastcannon fails to fire without a pin.
hopefully it doesn't break other stuff but for now it seems like were okay.

Reply:
you should make the bamboo syringe gun thing not require a pin